### ISSUE TITLE
Please support aspect ratio 20:13

### ISSUE THREAD

**Issue by nomeata** [josephwright/beamer#497]
Guy Steele argues that when you don’t know whether to expect a 4:3 or a 16:9 projector, [you should use 20:13 as the aspect ratio](https://www.youtube.com/watch?v=7HKbjYqqPPQ).

But `beamer` supports only a fixed set of aspect ratios via the `aspectratio=` options, and 20:13 is not one of them.

Could it be added?

**Comment by SFr682k** [comment]
I just tried to find page dimensions for frames with aspect ratio 20:13, but as close as possible to page dimensions already implemented in `beamer`.

Results:  
16cm×10.4cm, 14.77cm×9.6cm, 14cm×9.1cm

Then I took a look at the documentation and got aware of `aspectratio=149`, setting the page dimensions to 14cm×9.0cm. It wastes 14.29% of available space on 4:3 screens (20:13 wastes 13.33%), but only 12.50% on 16:9 screens (20:13 wastes 13.46% here).


#### TL;DR:   
For now, there are two possible workarounds:
a) Use `aspectratio=149`  
b) Set the frame size manually:

    \documentclass{beamer}
    \geometry{paperwidth=14cm,paperheight=9.1cm}
    \begin{document}
      \frame{A frame with aspect ratio 20:13}
    \end{document}

On a long-term basis, frames with a “real” 20:13 aspect ratio would be preferable.

**Comment by NMarkgraf** [comment]
Just one tiny little question: Why did you choose 14cmx9.1cm and not 16cmx10.4cm?

**Comment by josephwright** [comment]
@NMarkgraf  I guess as there is an overall scaling to worry about

**Comment by SFr682k** [comment]
>  I guess as there is an overall scaling to worry about

That’s the point why I would prefer a page size of 14cm×9.1cm over 16cm×10.4cm.

Beamer uses a font size of 11pt at _all_ page sizes. Therefore, as you increase the page size, the font’s size on the screen – and also the legibility of text on the slides – decreases, since larger slides are magnified less than smaller slides.   
So, at a constant font size, larger slides are harder to read.

**Comment by NMarkgraf** [comment]
Still open? - Why?
